lib/battleship.rb

```diff
@@ -28,19 +40,59 @@ def start
       # puts result of computer's shot
       # check if all user's ships are sunk
       # if yes, change @game_over to "computer"
-    end
-
-    end_game
-  end
+  #   end
+  #
+  #   end_game
+  # end
 
   def setup
     # set up two boards
+    players = [@user, @computer]
     # @boards << computer board
     # @boards << user board
     # ask @computer to place ships
     # ask @user to place ships
+
+    players.each do |player|
+      player.ships.each do |ship|
```

Comment:
Let's place the ships as a method inside the Player class. That way, our Computer can place its own ships too, without having to put that in the main Battleship class.